app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,42 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        From the home screen, swipe down to display the favorites panel and swipe up to hide it.
+        Swipe 🡇 from the home screen to show all apps, and 🡇 to go back.
         \n
-        \nIf you swipe up when the favorites panel is hidden, it will display the complete list of applications.
-        Swipe down from the top of this list to come back to the home screen.
+        \nSwipe 🡇 from the home screen to show favorites, and 🡅 to go back.
         \n
-        \nPress the button below the favorites panel to display the menu of the launcher.
+        \nClick 🔧 when showing favorites to open the menu.
         \n
-        \nIf you hold your finger a few moments on an application or a folder, you will be offered to access its settings.
+        \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
-        The notification allows to display a popup with favorites above other applications when you leave the home screen.
-        In most cases, it should work directly and without issues.
-        \n
-        \nHowever, in some special cases it may bring back to the home screen instead of being displayed over other applications.
-        \nIf you are facing this issue, there are two things you can try:
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
-        \n
-        \nTo not be intrusive, the notification icon is transparent and low priority.
-        With some applications having even lower priorities, this may result in an empty space in the status bar.
-        \n
-        \nFinally, if you change the default launcher, you may still see the notification.
-        To hide it, you can open Discreet Launcher (no need to be the default launcher) and disable the notification in the settings.
+        Shows a popup of favorites above other apps when you leave the home screen.

```

Comment:
```suggestion
        Shows a popup of favorites above other apps when you leave the home screen.
        \n
        \nUnintrusively, the notification icon is transparent and of low priority.
        With some apps having even lower priorities, this may result in empty space in the status bar.
        \n
        \nIf the Discreet Launcher notification is still there after changing to a different launcher, you can
        turn it off in the Discreet Launcher settings.
```
*(putting the comment back here, I wrote it in the commit sorry)*
Why remove the paragraphs about the transparent icon and the case of switching to another launcher?
I think there were useful and now there is only one line in the notification section...

Reply:
My idea was to have no troubleshooting for stuff that should either work or is designed that way.
I assumed it goes away after rebooting the device?
The actual explanations become clearer if there aren't caveats thrown in.

Reply:
Yes I agree that the troubleshooting part (clear apps and/or reboot) can be removed as it was added due to past issues which have now be solved over time.
But I was talking about the few lines below that we recently simplified (29 to 33 in the original file).

> I assumed it goes away after rebooting the device?

Yes if you change the default launcher and reboot, everything will restart from a clean page. But I don't think many users reboot after changing their default launcher, that sounds quite overkill (in fact, it is more probable that they would uninstall Discreet Launcher, which also removes any trace of the notification).

Regarding the empty space in the status bar, it depends on the other applications started at the same time and the priority of their notifications. Rebooting the phone would indeed close these other apps, but that will not change the way Android manages the order of notifications in the status bar.

Reply:
For the favorites notification: It is either a feature rather than a bug, or users aren't going to find it explained in the app they just uninstalled.
For everyone else it complicates getting to terms with the actual app, and sends a bit of a mixed message in suggesting users could remove the launcher.

For the missing space: This is a minor issue, but adding extra text for translation and visual comprehension of what could otherwise be a fully translated and simple description is more of an issue IMO.

I didn't read it at all because there was too much text, and explaining any of it easily or translating that is not likely to be better I think. Could be reintroduced if it seems to be missing or someone complains.
What I think is that simple and functional through and through sends a message not otherwise conveyed.

Reply:
Ok, let's give it a try like this!